### PR TITLE
Add @owang bio file

### PR DESCRIPTION
**Pull request by oliviagwang**
This Pull Request adds a bio file for @owang as requested #17.

Please take a :eyes: at this and let me know if it needs anything.

Have a good day! :smile:

### CODE REVIEW

**Comment by oliviagwang** [comment]
This looks :ok_hand: to me.  Maybe there is room for improvement though.

**Comment by agtaylor-mit** [comment]
Hmm.... It looks like I'm the one that needs to review your commit :smiling_imp: @oliviagwang

File: docs/owang-bio.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## @owang
+
+**Hometown:** Boston, MA
+
+**Hobbies:** Learning about Git and GitHub
+
+**Favorite Food:** Sushi
```
Comment (agtaylor-mit) [inline docs/owang-bio.md:7]: What other foods to you like? Sushi is awesome btw :+1:  @oliviagwang

Comment (oliviagwang) [inline docs/owang-bio.md:7]: I like pizza too :smiley:

Comment (agtaylor-mit) [inline docs/owang-bio.md:7]: Cool lets add that in there too @oliviagwang

**Comment by agtaylor-mit** [comment]
:+1:

**Comment by oliviagwang** [comment]
:+1:

**Pull request merged**